src/appleseed/foundation/meta/tests/test_aabb.cpp

```diff
@@ -384,7 +373,7 @@ TEST_SUITE(Foundation_Math_AABB)
             Vector3d(-1.0, -2.0, -3.0),
             Vector3d(4.0, 5.0, 6.0));
 
-        EXPECT_FEQ(Vector3d(5.0, 7.0, 9.0), bbox.extent());
+        EXPECT_EQ(Vector3d(5.0, 7.0, 9.0), bbox.extent());
```

Comment:
Did you run the unit tests? I'm expecting them to fail now that you replaced `EXPECT_FEQ()` by `EXPECT_EQ()`. Revert these changes.